apps/frontend/src/components/fap/Members/FapMembers.tsx

```diff
@@ -372,7 +379,21 @@ const FapMembers = ({
       </Grid>
       {isUserOfficer && (
         <Grid>
-          <div style={{ textAlign: 'right' }}>
+          <div
```

Comment:
I guess instead of using `<Grid>` and `<div>` here you can just use `<Stack>` (https://mui.com/material-ui/react-stack/) component and simplify the UI.